index.html

```diff
@@ -258,6 +258,56 @@ <h2>Myself</h2>
             </ul>
           </div>
         </div>
+
+        <footer>
+          <form
+            action="https://formspree.io/f/mgedkqkj"
+            method="post"
+            class="contact-form"
+            autocomplete="off"
+          >
+            <h2>Contact Me</h2>
+            <div class="parag">
+              <p>
+                If you have an application you are interested in developing, a
+                feature that you need built or a project that needs coding.I'd
+                love to help with it.
+              </p>
+            </div>
+            <div class="data1">
+              <input
+                name="Name"
+                type="text"
+                placeholder="Yusuf"
+                required
+                maxlength="30"
+              />
+              <br />
+              <input
+                name="email"
+                type="email"
+                id="email"
+                placeholder="yusufnviiri@gmail.com"
+                required
```

Comment:
-done the required changes on the README.md file but they don't reflect on the README.md file in the remote repo since it is already merged to the main branch and the changes only appear on the contact-form branch under review...kindly advise whether I should go on and create a pull request